clangd/ClangdLSPServer.cpp

```diff
@@ -65,10 +66,18 @@ void ClangdLSPServer::onInitialize(Ctx C, InitializeParams &Params) {
             {"referencesProvider", true},
             {"workspaceSymbolProvider", true},
         }}}});
+  std::vector<Path> ExclusionList;
+
+  if (Params.initializationOptions.hasValue()) {
```

Comment:
If you introduce setConfiguration (see comment below), I don't think you'll need this block.